packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts

```diff
@@ -65,12 +135,32 @@ export class LogContext {
   }
 
   public parseLogEvent(logLine: string): DebugLogState {
-    if (!this.state) {
+    if (this.state === undefined) {
       return new LogEntryState();
     }
+    if (logLine.startsWith(EVENT_EXECUTE_ANONYMOUS)) {
+      this.execAnonMapping.set(
+        this.execAnonMapping.size + 1,
+        this.logLinePosition + 1
+      );
+    }
     const fields = logLine.split('|');
     if (fields.length >= 3) {
       switch (fields[1]) {
```

Comment:
Some new events that add a frame, remove a frame, or update the line value of a frame.